scripts/assets/file_content_validation_checks.textproto

```diff
@@ -0,0 +1,5 @@
+file_content_checks {
+  filename_regex: ".+?.kt"
+  prohibited_content_regex: "^import .+?support.+?$"
+  failure_message: "AndroidX should be used instead of the support library"
+}
```

Comment:
add eof

Reply:
Also, we can name something which resemble more on prohibiting the use of support library.

Reply:
Done.